Frameworks/CoreText/CTFont.mm

```diff
@@ -225,10 +283,8 @@ CFTypeRef CTFontCopyAttribute(CTFontRef font, CFStringRef attribute) {
 /**
  @Status Interoperable
 */
-float CTFontGetSize(CTFontRef font) {
-    float ret = [(UIFont*)font pointSize];
-
-    return ret;
+CGFloat CTFontGetSize(CTFontRef font) {
+    return font->_size;
```

Comment:
Actually no I changed my mind. If the caller is using this very plain C-style function on a null pointer, I think they probably deserve the AV that they get?

---
In reply to: [78227303](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78227303) [](ancestors = 78227303,78206516)